src/App/Models/WindowsEzStartup.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace App.Models
```

Comment:
Models project should just contain data / aka POCO (plain ol class object)

A model shouldn't have methods in general, unless they are just parsing / manipulating the data or handling serialization of it

Reply:
I'd put this in the App project